Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Overrides/PrefabOverrideHandler.cpp

```diff
@@ -8,13 +8,34 @@
 
 #include <AzToolsFramework/API/ToolsApplicationAPI.h>
 #include <AzToolsFramework/Prefab/Overrides/PrefabOverrideHandler.h>
+#include <AzToolsFramework/Prefab/PrefabDomUtils.h>
 #include <AzToolsFramework/Prefab/PrefabSystemComponentInterface.h>
 #include <AzToolsFramework/Prefab/Undo/PrefabUndoRevertOverrides.h>
 
  namespace AzToolsFramework
 {
     namespace Prefab
     {
+        namespace Internal
+        {
+            // Helper to differentiate between adding/removing
+            // an entity object or adding/removing an entity property
+            bool IsDirectEntityPatch(PrefabDomConstReference patch)
+            {
+                auto pathMember = patch->get().FindMember("path");
+                if (pathMember != patch->get().MemberEnd())
+                {
+                    AZ::Dom::Path path(pathMember->value.GetString());
+                    if (path.Size() > 1)
+                    {
+                        return (path[path.Size() - 2] == PrefabDomUtils::EntitiesName);
```

Comment:
This will map any override that has the second to last word as 'Entities'. If a component in the future chooses to serialize on of its fields to 'Entities', we will get a false positive. 

Instead of doing it this way, a better way imo would be to do something like:
```
if (get overrides present at path /PathFromFocusedPrefab/Entities/TheCurrentEntityAlias)
{
   if(only 1 override is present)
   {
       if( type is 'add')
       {
          // show the 'plus' icon
       }
       else
       {
           // show 'dot' icon
       }
   }
   else
   {
       // show 'dot' icon
   }
}

```

Feel free to reach out to me if you have any questions on the above approach

Reply:
Even with only one override present with an "add" or "remove" operation, it still doesn't determine whether we're adding an entity or editing an entity.